source/expressions.tex

```diff
@@ -4163,13 +4163,12 @@
 \pnum
 \indextext{expression!unary operator}%
 \indextext{operator!unary}%
-The unary \tcode{*} operator performs \defn{indirection}:
+The unary \tcode{*} operator performs \defn{indirection}.
 \indextext{dereferencing|see{indirection}}%
-the expression to which it is applied shall be a pointer to an object
-type, or a pointer to a function type and the result is an lvalue
-referring to the object or function to which the expression points. If
-the type of the expression is ``pointer to \tcode{T}'', the type of the
-result is ``\tcode{T}''.
+Its operand shall be a prvalue of type ``pointer to \tcode{T}'',
+where \tcode{T} is an object or function type.
+Application of the operator is an lvalue of type \tcode{T}
```

Comment:
Describing the expression as "Application of the operator" seems a bit unusual. We could make this parallel with the wording change below:
```suggestion
The operator yields an lvalue of type \tcode{T}
```